docs/apache-airflow/tutorial_taskflow_api.rst

```diff
@@ -177,6 +177,40 @@ is automatically set to true.
 Note, If you manually set the ``multiple_outputs`` parameter the inference is disabled and
 the parameter value is used.
 
+Adding dependencies to decorated tasks from regular tasks
+---------------------------------------------------------
+The above tutorial shows how to create dependencies between python-based tasks. However, it is
+quite possible while writing a DAG to have some pre-existing tasks such as BashOperator or FileSensor
```

Comment:
```suggestion
quite possible while writing a DAG to have some pre-existing tasks such as :class:`~airflow.operators.bash.BashOperator` or :class:`~airflow.sensors.filesystem.FileSensor`
```

Reply:
This will make it easy for users to go to the Operator / Sensor definition if they are not already aware about it